### PR TITLE
Portfolio

### PR DESCRIPTION
**Pull request by p3drac0**
I created a completely stunning and destructive portfolio

### CODE REVIEW

**Review by LuisFlor7 — APPROVED** [review]
tod bien

File: website/templates/portfolio.html
Comment (ScarzoJ) [inline website/templates/portfolio.html:1]: Format whole file

File: website/templates/portfolio.html [hunk 1]
```diff
@@ -1,2 +1,40 @@
...
+            </div>
+          </div>
+      </div>
+      <div class="col">
+        <div class="card" style="width: 100%;">
+            <img src="https://cdns-images.dzcdn.net/images/cover/6e59542dabf670bee95873bc958b1a88/0x1900-000000-80-0-0.jpg" class="card-img-top" alt="...">
+            <div class="card-body">
+              <h5 class="card-title">No puedo contener mi ira</h5>
+              <p class="card-text">Quiero quemar cosas con un desodorante y un mechero</p>
+
```
Comment (ScarzoJ) [inline website/templates/portfolio.html]: Remove newlines from the document

**Review by ScarzoJ — CHANGES_REQUESTED** [review]

**Review by battanair — COMMENTED** [review]
Maravilla

**Review by battanair — DISMISSED** [review]
boh

**Review by LuisFlor7 — DISMISSED** [review]

**Review by ScarzoJ — APPROVED** [review]

File: website/templates/portfolio.html
Comment (p3drac0) [reply website/templates/portfolio.html:1]: todo bien

File: website/templates/portfolio.html (hunk 1 above)
Comment (p3drac0) [reply website/templates/portfolio.html]: que si, que ya está

**Review by LuisFlor7 — APPROVED** [review]

**Pull request merged**